traitquestdev/kra.php

```diff
@@ -0,0 +1,182 @@
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-briefcase"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">Company</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-user"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">Profile</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s sideBarSelected">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-user"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">KRA</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-stats"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">KPI</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-user"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">People</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-cog"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">Settings</h4>
+				</li></a>
+				<a href="#"><li class="padding-topbottom-s">
+					<h4 class="display-block display-inline padding-left10px"><i class="glyphicon glyphicon-off"></i></h4>
+					<h4 class="display-block display-inline padding-left10px">Logout</h4>
+				</li></a>
+
+			</ul>
+		</div>
+	</div>
+
+
+	<div class="clear col-lg-10 col-lg-offset-2 col-md-10 col-md-offset-2 padding-topbottom-xs grey95-bg">
+
+			<a href="http://traitquest.com/">
+			<img id="logo-image" src="images/logo.png" /></a>
+
+		<i id="openMobileNav" class="glyphicon glyphicon-menu-hamburger
+			hidden-lg hidden-md col-sm-2 col-xs-2 float-right
+			display-block display-inline padding-topbottom-s text-right fontsize-l cursor-pointer"></i>
+
+
+	</div>
+
+
+	<div class="clear col-lg-10 col-lg-offset-2 col-md-10 col-md-offset-2 white-bg">
+
+			<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10 padding-topbottom-s">
+				<h3>KRA</h3>
+			</div>
+
+	</div>
+
+	<div class="col-lg-10 col-lg-offset-2 col-md-10 col-md-offset-2 col-sm-12 col-xs-12 grey95-bg padding-bottom-s"><!content goes here>
+		<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 margin-topbottom-s padding-leftright-s padding-bottom-s border-top-grey white-bg"><!1stwindow>
+
+			<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 margin-topbottom-s padding-leftright-s padding-bottom-s">
+				<div class="img-wrap">
+					<h4 class="addobjective"><a href="#"><i class="black glyphicon glyphicon-plus-sign"></i></a></h4>
+
```

Comment:
there will only be one date dropdown for both month and year. Refer to http://www.developerji.com/ImagesUpload/DateTime20130910.png. I want a calendar icon at the end of the dropdown which is a button. Clicking it will show the calendar. This applies to all the date dropdown.